src/website/app/demos/Flex/examples/FlexItem/flexProps.js

```diff
@@ -0,0 +1,36 @@
+/* @flow */
+import { createStyledComponent } from '../../../../../../library/styles';
+import _FlexItem from '../../components/FlexItem';
+import Box from '../../../Box/components/Box';
+import Flex from '../../components/Flex';
+
+// Resetting some styles applied by _FlexItem's boxStyles
+const FlexItem = createStyledComponent(
+  _FlexItem,
+  ({ alignItems, flex, justifyContent }) => ({
+    alignItems: alignItems || 'flex-start',
+    display: flex ? 'flex' : 'block',
+    justifyContent: justifyContent || 'flex-start'
+  })
+);
+
+export default {
+  id: 'flex-props',
+  title: 'Flex Props',
+  description: `FlexItem can optionally style the [Flex](/components/flex)
+component with the \`flex\` prop, allowing it to accept any of
```

Comment:
You may want to call this out more clearly; maybe "The `flex` prop renders the `FlexItem` as a `Flex` container... "

Reply:
Updated like so: https://github.com/mineral-ui/mineral-ui/blob/f44b50e937a942bcb33651b8891e64536e34a8ba/src/website/app/demos/Flex/examples/FlexItem/flexProps.js#L20-L23

Reply:
I also updated all of the box props examples to use the same "composes" language: https://github.com/mineral-ui/mineral-ui/blob/f44b50e937a942bcb33651b8891e64536e34a8ba/src/website/app/demos/Flex/examples/FlexItem/boxProps.js#L8-L11

(Only FlexItem's has that second sentence.)